Lib/Timer.cpp

```diff
@@ -223,21 +223,13 @@ void Lib::Timer::syncClock()
 {
   if(s_initGuarantedMiliseconds==-1) {
     //we're unable to sync clock as we weren't able to obtain number of ticks in the beginning
-    bool reportedProblem = false;
-    if(!reportedProblem) {
-      reportedProblem = true;
-      cerr << "cannot syncronize clock as times() initially returned -1" << endl;
-    }
+    cerr << "cannot synchronize clock as times() initially returned -1" << endl;
```

Comment:
I think this code is stopping is from printing the error more than once but, for this to work, we should make reportedProblem static.

Reply:
Ah, that makes more sense. I thought it was a refactor-gone-wrong. I'll static it.